rasa/nlu/training_data/training_data.py

```diff
@@ -106,13 +106,17 @@ def __hash__(self) -> int:
     def sanitize_examples(examples: List[Message]) -> List[Message]:
         """Makes sure the training data is clean.
 
-        removes trailing whitespaces from intent annotations."""
+        Remove trailing whitespaces from intent and response annotations and drop duplicate examples."""
 
         for ex in examples:
             if ex.get("intent"):
                 ex.set("intent", ex.get("intent").strip())
+
             if ex.get("response"):
                 ex.set("response", ex.get("response").strip())
+
+        examples = list(OrderedDict.fromkeys(examples))
+
         return examples
```

Comment:
Let's change this for `return list(OrderedDict.fromkeys(examples))`

Reply:
done